src/components/ChooseOfferForm/ChooseOfferForm.tsx

```diff
@@ -69,23 +72,25 @@ const ChooseOfferForm: React.FC<Props> = ({
     return null;
   };
 
-  const OfferBox: OfferBox = ({ offer, radioKey, radioValue, title, ariaLabel, secondBenefit, periodString }) => (
+  console.info(values.offerId);
```

Comment:
Make sure we take this out before merging

Reply:
Removed, thanks!